charts/fluent-bit/templates/scc.yaml

```diff
@@ -1,12 +1,14 @@
-{{- if and .Values.openShift.enabled .Values.openShift.securityContextConstraints.create }}
-apiVersion: security.openshift.io/v1
+{{- if and .Values.openShift.securityContextConstraints.create }}
```

Comment:
```suggestion
{{- if and .Values.openShift.enabled .Values.openShift.securityContextConstraints.create }}
```

Reply:
@stevehipwell but why do you think there is ".Values.openShift.enabled" required? It, logically, does nothing - only adds one more on/off switch. If user will turn it "on" - nothing will change, without ".Values.openShift.securityContextConstraints.*" changing

Reply:
Because it makes logical sense to only evaluate for OpenShift if it's enabled, this is a standard pattern for Helm charts. The `openShift.enabled` value is already part of the spec and by removing it you've made the configuration more complex.